Web_Development/Javascript/Topics/6.Hoisting/README.md

```diff
@@ -0,0 +1,258 @@
+# Hoisting in Javascript
+
+### Introduction
+
+In this tutorial I will discuss the important concept of Hoisting in Javascript. Before deep diving into it let's see a small intro.
+>Hoisting is a mechanism in javascript where the variable and function declarations are moved to the top of their respective scopes before the code execution phase.
+
+Obviously now you must have many doubts after reading the intro. So let's simplify it with an example.  
+Did you ever notice that you can call a function even before it is declared ? 
+```js
+    console.log(sayHi())
+    
+    function sayHi(){
+        console.log("Hi Girlscript!!")
+    }
+    // The result of the code is : Hi Girlscript!!
+```
```

Comment:
You don't need to indent the code-blocks. Please get rid of the indentation.

Reply:
Ok I am removing the left indentations

Reply:
@paramsiddharth Added a commit to remove the indentation as per suggestions.